dotnet/src/Extensions/Planning.ActionPlanner/ActionPlanner.cs

```diff
@@ -230,7 +231,14 @@ private void PopulateList(StringBuilder list, IDictionary<string, List<FunctionV
             foreach (FunctionView func in skill.Value)
             {
                 // Function description
-                list.AppendLine($"// {AddPeriod(func.Description)}");
+                if (func.Description != null)
+                {
+                    list.AppendLine($"// {AddPeriod(func.Description)}");
+                }
+                else
+                {
+                    list.AppendLine($"// This is a description of {func.SkillName}.{func.Name}.");
```

Comment:
you know, we could consider throwing an error here. Using the planner with functions without a description is bound to go badly, and I think it would help leading developers in a direction where functions must be described to be planner-compatible.

Reply:
This is an unfortunate reality of OpenAPI definitions.

Reply:
we receive a lot of feedback about the planner not doing the right thing, we should at least log a warning. Also we can save some tokens here making the comment shorter, e.g.
```
// Function {func.SkillName}.{func.Name}
```